common/static/common/js/components/BlockBrowser/data/api/client.js

```diff
@@ -21,7 +21,7 @@ export function buildQueryString(data) {
 }
 
 export const getCourseBlocks = courseId => fetch(
-  `${COURSE_BLOCKS_API}/?${buildQueryString({
+  `${COURSE_BLOCKS_API}?${buildQueryString({
```

Comment:
note that `const COURSE_BLOCKS_API = '/api/courses/v1/blocks/'` so there is already a slash